Dockerfile

```
@@ -1,21 +1,48 @@
-COPY --from=builder --chown=www-data /tmp/rust/src/github.com/stockwayup/http2/target/release/http2 /bin/http2
-
-RUN chmod +x /bin/http2
-
-USER www-data
-
-CMD ["/bin/http2"]
+# syntax=docker/dockerfile:1.5
+
+# Base stage with all build dependencies
+FROM rust:1.89-bookworm AS chef
+RUN cargo install cargo-chef --locked && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)